.github/prompts/freshness-tier1.prompt.md

```diff
@@ -12,5 +12,6 @@ how fresh and up to date it is. Apply the following:
 5. Ensure all the components formatted correctly.
 6. Improve the SEO keywords.
 7. If you find numbered lists, make sure their numbering only uses 1's.
+8. Keep line length to a maximum of 90 characters, and do not add a space character at the end of lines.
```

Comment:
It should be 80 to be conform with what we currently have across all markdown. This is also part of our conventions: https://github.com/docker/docs/blob/331240357cedab35b722d321b46d96aa241e1c09/content/contribute/file-conventions.md?plain=1#L66

Reply:
Ah yes indeed some pages are not aligned. Wonder if we could have a way to validate this with a linter.